README.md

```diff
@@ -697,5 +711,6 @@ That being said, I have created my [own version of the ULX Module](https://steam
 - Alex, Rhettg32, and Bartez from the [Lonely Yogs](https://lonely-yogs.co.uk/) Discord for the idea for the "Flu Season" event
 - Bartez from the [Lonely Yogs](https://lonely-yogs.co.uk/) Discord for the idea for the "NO NERD HEALING" event
 - [Guardian954](https://steamcommunity.com/id/guardianreborn) for the initial "Communism! Time to learn how to share..." event and
```

Comment:
I would split this into two lines. End Guardian's line after "event" and then tweak your line to thank you for converting Communism and adding Gravity

Reply:
Done